spec/unit/verifier_spec.rb

```diff
@@ -167,6 +167,42 @@
     expect(result.errors.first).to eq "Expected file name: awesomelib.2013-09-08.manifest"
   end
 
+  context "one invalid topic" do
+    it "detects invalid topics" do
+      handler = ManifestHandler.new settings
+      handler.read_remote
+      verifier = Verifier.new settings
+
+      manifest = handler.manifest("awesomelib")
+      expect(verifier.verify(manifest).valid?).to be true
+
+      manifest.topics = ["Invalid"]
+
+      result = verifier.verify(manifest)
+
+      expect(result.valid?).to be false
+      expect(result.errors).to include "Invalid topics: Invalid. Valid topics are API,Artwork,Bindings,Communication,Data,Desktop,Development,Graphics,Logging,Mobile,Multimedia,Printing,QML,Scripting,Security,Text,Web,Widgets"
```

Comment:
It would be better to put the names of the topics in single quotes, so that for the reader of the message it's clear what the name of the topic is and what the text of the error message around that is. I would also put a space after each comma to give more natural spelling.